### ISSUE TITLE
Crash on ED launch (Colore problem?)

### ISSUE THREAD

**Issue by billhedge** [poveden/EliteChroma#92]
## Steps to Reproduce

1. Start EliteChroma
2. Start ED from launcher
3. EliteChroma icon disappears from systray

## Event Log info

Application: EliteChroma.exe
CoreCLR Version: 4.700.20.56602
.NET Core Version: 3.1.11
Description: The process was terminated due to an unhandled exception.
Exception Info: System.AggregateException: One or more errors occurred. (Call to native Chroma SDK function Init failed with error: Unknown: Unknown. (1647))
 ---> Colore.Native.NativeCallException: Call to native Chroma SDK function Init failed with error: Unknown: Unknown. (1647)
 ---> System.ComponentModel.Win32Exception (1647): The update is not applied to this product.
   --- End of inner exception stack trace ---
   at Colore.Native.NativeApi.InitializeAsync(AppInfo info)
   at Colore.Implementations.ChromaImplementation.InitializeAsync(AppInfo info)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Colore.Implementations.ChromaImplementation..ctor(IChromaApi api, AppInfo info)
   at Colore.ColoreProvider.CreateAsync(AppInfo info, IChromaApi api)
   at EliteChroma.Core.Windows.WinChromaFactory.CreateAsync() in D:\Dev\GitHub\poveden\EliteChroma\src\EliteChroma.Core.Windows\WinChromaFactory.cs:line 35
   at EliteChroma.Core.ChromaController.ChromaStart() in D:\Dev\GitHub\poveden\EliteChroma\src\EliteChroma.Core\ChromaController.cs:line 180
   at EliteChroma.Core.ChromaController.RenderEffect() in D:\Dev\GitHub\poveden\EliteChroma\src\EliteChroma.Core\ChromaController.cs:line 243
   at EliteChroma.Core.ChromaController.GameState_Changed(Object sender, ChangeType e) in D:\Dev\GitHub\poveden\EliteChroma\src\EliteChroma.Core\ChromaController.cs:line 216
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__139_1(Object state)
   at System.Threading.QueueUserWorkItemCallbackDefaultContext.Execute()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
## Environment

- `EliteChroma` version: 1.10.0
- .NET Core version: 5.0.1
- Operating System: Win 10
- Synapse version: 3.5.1215.121019

## Additional Context

This has been happening for about a month and only started after I updated Synapse. Based off of the stack trace and exception, it appears that something changed in the SDK that Colore can't deal with. This may be something the the Colore folks need to look at, but I thought I'd send it to you first, just in case.

**Comment by poveden** [comment]
@billhedge Is it still failing for you? I've done some tests, but cannot reproduce your issue.

**Comment by billhedge** [comment]
Sadly, yes. It's still happening. I just reinstalled EliteChroma to see if that had any affect, but it did not. I suppose I could reinstall Synapse, if you think that might help. I have the most recent update of that installed currently (from 4 February).

*snip*

I left the above intact instead of simply erasing it and replacing it with the following.

I just uninstalled and reinstalled all of the Razer s/w and now it works. Ugh. I should have done that to begin with. Sorry!
